addons/account/models/account_move.py

```diff
@@ -1309,7 +1306,7 @@ def _compute_amount(self):
             move.amount_total_signed = abs(total) if move.move_type == 'entry' else -total
             move.amount_residual_signed = total_residual
 
-            currency = len(currencies) == 1 and currencies.pop() or move.company_id.currency_id
+            currency = len(currencies) == 1 and currencies or move.company_id.currency_id
```

Comment:
```suggestion
            currency = len(currencies) == 1 or move.company_id.currency_id
```

Reply:
Why... This is wrong... My condition was correct

Reply:
True, I'm tired maybe